### ISSUE TITLE
Would it be possible to load SVG textures? 

### ISSUE THREAD

**Issue by jarben** [ecomfe/echarts-gl#38]
Would be nice if SVG files can be loaded as textures, specially globle:environment so that stars are not blurred:) Would this be possible?

**Comment by pissang** [comment]
You can use a higher resolution star environment image. But since WebGL can only use pixels in GPU. It's not really matter if the image source is SVG or PNG or JPEG.

But still you can have a try:)

**Comment by jarben** [comment]
I see - thanks for your answer.

**Comment by jarben** [comment]
One more question - would it be possible to use a canvas reference similarly to the baseTexture property? In that case, environment can be also generated which would save a request to a high res image + it can potentially animate stars etc:)

**Comment by pissang** [comment]
Yes. You can even use a ECharts instance in the `environment` property. And texture will be automatically updated when the canvas of ECharts is refreshed.

**Comment by jarben** [comment]
Very cool! Will play with that.
